File: 2234/2234.js
var lines = ['10 90', '840 11', '1 50', '34 1000', '35 1000'];

lines.forEach(f);

function f(v){
    var h, p;
    h = parseInt(v.split(' ')[0]);
    p = parseInt(v.split(' ')[1]);
    var r = h/p;
    r = 1.994;
    console.log(r.toFixed(2));
}

File: 2234/2234.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.lang.*;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
       BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
       String linha;
       while ((linha = br.readLine()) != null)
       {
           double a = Integer.parseInt(linha.split(" ")[0]);
           double b = Integer.parseInt(linha.split(" ")[1]);
            
           System.out.printf("%.2f\n",a/b);
       }
 
    }
 
}